server/src/main/java/com/lunchtime/models/Users.java

```diff
@@ -0,0 +1,42 @@
+package com.lunchtime.models;
+import lombok.Data;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+import java.util.List;
+
+
+@Entity
+@Table(name = "users")
```

Comment:
You should use singular noun in table name
https://rv-054java.slack.com/archives/CUKLK9279/p1583759766037900